app/scripts/services/journalUploadService.js

```diff
@@ -0,0 +1,139 @@
+        filterAccountTransferErrors: 'yes',
+        filterAccountTransfersByExpenseType: '',
+        filterAccountTransfersByPaymentType: '',
+        filterPayment: '',
+        filterRegType: '',
+        includeAccountTransfers: true,
+        includeCheckedin: 'yes',
+        includeWithdrawn: 'yes',
+        includeIncomplete: 'yes',
+        primaryRegistrantOnly: true,
+      },
+    ) => {
+      const defer = $q.defer();
+      $rootScope.loadingMsg = 'Loading Registrations';
+
+      $http
+        .get(`${path(conferenceId)}/registrations`, {
+          params: queryParameters,
+        })
+        .then(response => {
+          $rootScope.loadingMsg = '';
+          defer.resolve(response.data);
+        })
+        .catch(() => {
+          $rootScope.loadingMsg = '';
+          defer.reject();
+        });
+
+      return defer.promise;
+    };
+
+    this.getAllAccountTransferReports = conferenceId => {
+      const defer = $q.defer();
+      $rootScope.loadingMsg = 'Loading Reports';
+
+      $http
+        .get(`${path(conferenceId)}/account/transfer/reports`)
+        .then(response => {
+          $rootScope.loadingMsg = '';
+          defer.resolve(response.data);
+        })
+        .catch(() => {
+          $rootScope.loadingMsg = '';
+          defer.reject();
+        });
+
+      return defer.promise;
+    };
+
+    this.getAccountTransferReport = url => {
+      const defer = $q.defer();
+      $rootScope.loadingMsg = 'Loading Report';
+
+      $http
+        .get(url)
+        .then(response => {
+          $rootScope.loadingMsg = '';
+          defer.resolve(response.data);
+        })
+        .catch(() => {
+          $rootScope.loadingMsg = '';
+          defer.reject();
+        });
+
+      return defer.promise;
+    };
+
+    this.getAccountTransferData = data => {
+      return _.flatten(
+        data.registrations.reduce((result, registrant) => {
+          return [
+            ...result,
+            registrant.accountTransfers.filter(
+              accountTransfer => !accountTransfer.error,
+            ),
+          ];
+        }, []),
+      );
+    };
+
+    this.getAccountTransferDataWithErrors = data => {
+      return _.flatten(
+        data.registrations.reduce((result, registrant) => {
+          return [
+            ...result,
+            registrant.accountTransfers.filter(
+              accountTransfer => accountTransfer.error,
+            ),
+          ];
+        }, []),
+      );
+    };
+
+    this.submitAccountTransfers = accountTransfers => {
+      $rootScope.loadingMsg = 'Submitting account transfers';
+      // Filter out included key from account transfer object
+      const filteredAccountTransfers = accountTransfers.map(accountTransfer => {
+        return Object.keys(accountTransfer)
+          .filter(key => key !== 'included')
+          .reduce((obj, key) => {
+            obj[key] = accountTransfer[key];
+            return obj;
+          }, {});
+      });
```

Comment:
```suggestion
      const filteredAccountTransfers = accountTransfers.map(({ included, ...accountTransfer }) => accountTransfer);
```

Reply:
I had to add the babel plugin ```transform-object-rest-spread``` and then update the eslint rule: 
```
"no-unused-vars": [
      "error",
      { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }
    ],
```

but it works now 🙂

Reply:
But it breaks Karma 😞